recipes/karilang-kernel/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "karilang-kernel" %}
+{% set version = "0.1.0" %}
+{% set karilang_version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/Vipul-Cariappa/KariLang-Kernel/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: de75bbe9a533170ff6889b204adb3b398823076a15173579cb7b6e2b7b1651f4
+    folder: karilang-kernel
+  - url: https://github.com/Vipul-Cariappa/KariLang/archive/refs/tags/v{{ karilang_version }}.tar.gz
+    sha256: 9e22f9bf4af9d70688cbb80856cfe358b9d1956face86a852696ab340300cb0c
+    folder: karilang-kernel/src/KariLang
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bison  # [unix]
+    - flex  # [unix]
+    - winflexbison  # [win]
+  host:
+    - xeus-zmq >=1.0.2,<2.0
+    - nlohmann_json
+    - cppzmq
+    - xtl
+    - xeus
+    - zlib
+    - zeromq
+  run:
+    - python
```

Do we need `python` in `host` too?

```suggestion
    - python
  run:
    - python
```